### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by Aishanur** [matifandy8/NeoBrutalismCSS#8] [Q&A] [answered]
Hello, I really appreciate the work you did, and I am interested in using these components in my portfolio. What is the license for this project?

**Comment by matifandy8** [comment, marked as answer]
Hi ,This project does not have a formal license, so you are free to use it without any restrictions. Feel free to incorporate it into your portfolio as you see fit.
I appreciate your feedback and am always open to any further comments or suggestions you may have.
